### PR TITLE
Enumerable testing

### PR DESCRIPTION
**Pull request by Emmyn5600**
# This pull request contains the following changes: 

##### We tested the possible solutions to see if they're working properly.

### CODE REVIEW

File: spec/enumerable_tests.rb [hunk 1]
```diff
@@ -39,23 +39,23 @@
...
                                                                                     hash if index == 1
                                                                                   end)
     end
   end
 
   describe '#my_select' do
-    it 'return elements that have 6 letters' do
+    it 'returns elements that have 6 letters' do
       expect(array.my_select { |word| word if word.length == 6 }).to eq(%w[Orange Banana])
     end
```
Comment (Elbie-em) [inline spec/enumerable_tests.rb:69]: - [ ] Kindly test positive and negative scenarios, an example, in this case, could be adding a word to your array that has a length greater or less than 6 to create a negative scenario.

**Review by Elbie-em — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ 🛑

Hello @Emmyn5600 and @leolpaz, I would like to congratulate you on your great performance.👏🏿 Great work trying to implement all the requirements. This was a great effort. However, there are changes to be made kindly check below ⬇️

- [ ] Rubocop flags the error below when run, please fix this.
rubocop failure

- [ ] Kindly test for positive and negative scenarios. I have left a comment giving an example of how you can implement this using the my_select method
image

## Kindly apply the changes requested so that the project can be approved

## NOTE: 
> When you open a pull request and it is marked as invalid, kindly submit the same pull request after you have applied changes that a TSE requests. You should not create a new pull request.

### Feel free to contact me if you have any questions @Elbie-em

HAPPY CODING 💻

**Comment by Emmyn5600** [comment]
- We tested the negative scenario for the my_select method as it was requested

- We deleted the Rspec keyword  to correct linter errors shown by rubocop

**Review by george-shammar — CHANGES_REQUESTED** [review]
**### STATUS: CHANGES REQUIRED** :recycle:

It's a great job you have done there. Your tests are passing. Good job!!! :clap: :clap: :clap:

However, we need to make some changes:

- [ ] Negative text cases are still not been covered. You need to expand the scope of your test to cover negative cases. There should be a spec for the correct usage of the method and also a spec for the wrong usage of the method. E.g positive test case (it "returns all the numbers divisible by 2") E.g negative test case( it "does not return all the numbers divisible by 2"). 

[### **OPTIONAL**]

- [x] Update your README. I couldn't run your tests with `rspec spec/enumerable.rb` as directed in your README but I was able to run it with `rspec spec/enumerable_tests.rb`

Overall, you're doing well. Do not be tired, you're almost there. :muscle: :muscle:

Submit another CRR once you're done.

If you need further clarification, reach me on slack at @George Gbenle

Happy coding. :smiley:

**Review by bini-i — APPROVED** [review]
# Status: Approved :heavy_check_mark:     
### Hello team :wave: 

Thanks for working on the required changes. Congratulations your project is **approved** :tada: 
You can now merge to master and submit the project completion form

#### If you have any question kindly reach me on slack @Binyam Hailemeskel

### HAPPY CODING :computer: :blush: 

> #### Reviewed by Binyam Hailemeskel (Part-Time TSE)
> GitHub: [@bini-i](https://github.com/bini-i) 
> Twitter: [@binyamshewa](https://twitter.com/binyamshewa)
> LinkedIn: [LinkedIn](https://www.linkedin.com/in/binyam-hailemeskel-728048151/)
